docs/user-guide/src/baremetal/guide.md

```diff
@@ -178,44 +214,18 @@ This is a separate machine, e.g. your laptop or one of the servers, that has acc
 ## Create and apply cluster, controlplane and worker template
 
   ```bash
-  # The URL of the kernel to deploy.
-  export DEPLOY_KERNEL_URL="http://172.22.0.1:6180/images/ironic-python-agent.kernel"
-
-  # The URL of the ramdisk to deploy.
-  export DEPLOY_RAMDISK_URL="http://172.22.0.1:6180/images/ironic-python-agent.initramfs"
-
-  # The URL of the Ironic endpoint.
-  export IRONIC_URL="http://172.22.0.1:6385/v1/"
-
-  # The URL of the Ironic inspector endpoint.
-  export IRONIC_INSPECTOR_URL="http://172.22.0.1:5050/v1/"
-
-  # Do not use a dedicated CA certificate for Ironic API.
-  # Any value provided in this variable disables additional CA certificate validation.
-  # To provide a CA certificate, leave this variable unset.
-  # If unset, then IRONIC_CA_CERT_B64 must be set.
-  export IRONIC_NO_CA_CERT=true
-
-  # Disables basic authentication for Ironic API.
-  # Any value provided in this variable disables authentication.
-  # To enable authentication, leave this variable unset.
-  # If unset, then IRONIC_USERNAME and IRONIC_PASSWORD must be set.
-  export IRONIC_NO_BASIC_AUTH=true
-
-  # Disables basic authentication for Ironic inspector API.
-  # Any value provided in this variable disables authentication.
-  # To enable authentication, leave this variable unset.
-  # If unset, then IRONIC_INSPECTOR_USERNAME and IRONIC_INSPECTOR_PASSWORD must be set.
-  export IRONIC_INSPECTOR_NO_BASIC_AUTH=true
-
   # Export node image variable and node image hash varibale that we created before.
   # Change name according to what was downlowded from artifactory
   export IMAGE_RAW_URL=http://172.22.0.1/images/CENTOS_9_NODE_IMAGE_K8S_v1.27.1-raw.img
   export IMAGE_RAW_CHECKSUM=http://172.22.0.1/images/CENTOS_9_NODE_IMAGE_K8S_v1.27.1-raw.img.sha256sum
```

Comment:
Do we need these or can we remove them?

Reply:
Yes we can remove them. I double checked and there are few more variables that are missed I am adding those too.

Reply:
Thanks!